src/System.Collections/src/System/Collections/BitArray.cs

```diff
@@ -202,12 +211,15 @@ public BitArray(BitArray bits)
         =========================================================================*/
         public bool Get(int index)
         {
-            if (index < 0 || index >= Length)
+            if ((uint)index >= (uint)Length)
```

Comment:
It could be argued that Roslyn should be doing such optimizations (and it actually does for `switch` cases, e.g., when you have `case '0'`  to ? `case '9'` with fallthroughs, it generates the subtraction and unsigned compare).
But then again in a general comparison like here it could end up hurting performance if for example it's common for index to be negative...